signer/src/transaction_coordinator.rs

```diff
@@ -666,7 +684,7 @@ where
                 // to do that at the source when we receive the message.
 
                 if &msg.bitcoin_chain_tip != chain_tip {
-                    tracing::warn!(?msg, "concurrent signing round message observed");
+                    tracing::warn!(origin = %msg.signer_pub_key, "concurrent signing round message observed");
```

Comment:
suggestion: i might call this "sender" or something a little bit clearer as to what it is

Reply:
Ohh much better, will do.